pkg/apis/m3dboperator/v1/cluster.go

```diff
@@ -75,6 +78,14 @@ type ClusterSpec struct {
 	Resources Resources `json:"resources"`
 }
 
+// IsolationGroup defines the name of zone as well attributes for the zone configuration
+type IsolationGroup struct {
+	// Name
+	Name string `json:"name"`
+	// InstancesAmount defines the amount of instances
+	InstanceAmount int32 `json:"instanceAmount"`
+}
+
 // Resources defines CPU / Memory restrictions on pods
 type Resources struct {
```

Comment:
could you constrain the types for Memory and CPU limits (to something more specific than `string`)